mmv1/third_party/terraform/utils/tagslocation_operation.go

```diff
@@ -0,0 +1,65 @@
+package google
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+	"time"
+)
+
+type TagsLocationOperationWaiter struct {
+	Config    *Config
+	UserAgent string
+	CommonOperationWaiter
+}
+
+func (w *TagsLocationOperationWaiter) QueryOp() (interface{}, error) {
+	if w == nil {
+		return nil, fmt.Errorf("Cannot query operation, it's unset or nil.")
+	}
+	location := GetLocationFromOpName(w.CommonOperationWaiter.Op.Name)
+	if location != w.CommonOperationWaiter.Op.Name {
+		// Found location in Op.Name, fill it in TagsLocationBasePath and rewrite URL
+		url := fmt.Sprintf("%s%s", strings.Replace(w.Config.TagsLocationBasePath, "{{location}}", location, 1), w.CommonOperationWaiter.Op.Name)
+		return sendRequest(w.Config, "GET", "", url, w.UserAgent, nil)
+	} else {
+		url := fmt.Sprintf("%s%s", w.Config.TagsBasePath, w.CommonOperationWaiter.Op.Name)
+		return sendRequest(w.Config, "GET", "", url, w.UserAgent, nil)
+	}
+}
+
+func createTagsLocationWaiter(config *Config, op map[string]interface{}, activity, userAgent string) (*TagsLocationOperationWaiter, error) {
+	w := &TagsLocationOperationWaiter{
+		Config:    config,
+		UserAgent: userAgent,
+	}
+	if err := w.CommonOperationWaiter.SetOp(op); err != nil {
+		return nil, err
+	}
+	return w, nil
+}
+
+// nolint: deadcode,unused
```

Comment:
Can you please remove this line, as the function tagsLocationOperationWaitTimeWithResponse is used?